internal/uvm/create_wcow.go

```diff
@@ -105,25 +105,54 @@ func prepareConfigDoc(ctx context.Context, uvm *UtilityVM, opts *OptionsWCOW, uv
 		},
 	}
 
+	var registryChanges hcsschema.RegistryChanges
+	// We're getting asked to setup local dump collection for WCOW. We need to:
+	//
+	// 1. Turn off WER reporting, so we don't both upload the dump and save a local copy.
+	// 2. Set WerSvc to start when the UVM starts to work around a bug when generating dumps for certain exceptions.
+	// https://github.com/microsoft/Windows-Containers/issues/60#issuecomment-834633192
+	// This supposedly should be fixed soon but for now keep this until we know which container images
+	// (1809, 1903/9, 2004 etc.) this went out too.
+	if opts.ProcessDumpLocation != "" {
+		uvm.processDumpLocation = opts.ProcessDumpLocation
+		registryChanges.AddValues = append(registryChanges.AddValues,
```

Comment:
Wouldn't this panic since `AddValues` is nil?

Reply:
```go
package main

import (
	"fmt"

	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
)

func main() {
	var registryChanges hcsschema.RegistryChanges
	registryChanges.AddValues = append(registryChanges.AddValues, hcsschema.RegistryValue{
		Key: &hcsschema.RegistryKey{
			Hive: "System",
			Name: "ControlSet001\\Services\\WerSvc",
		},
		Name:       "Start",
		DWordValue: 2,
		Type_:      "DWord",
	},
		hcsschema.RegistryValue{
			Key: &hcsschema.RegistryKey{
				Hive: "Software",
				Name: "Microsoft\\Windows\\Windows Error Reporting",
			},
			Name:       "Disabled",
			DWordValue: 1,
			Type_:      "DWord",
		},
	)

	fmt.Println("hi")
}
```

This works fine